http-server/src/server.rs

```diff
@@ -136,7 +136,7 @@ impl Server {
 
 	/// Returns a `Vec` with all the method names registered on this server.
 	pub fn method_names(&self) -> Vec<String> {
```

Comment:
we could do `impl IntoIterator <Item = &'static str>` here instead to avoid allocations